commands/bot.go

```diff
@@ -56,3 +113,143 @@ func botCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func botUpdateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	user := getUserFromUserArg(c, args[0])
+	if user == nil {
+		return errors.New("unable to find user '" + args[0] + "'")
+	}
+	patch := model.BotPatch{}
+	username, err := cmd.Flags().GetString("username")
+	if err == nil && cmd.Flags().Changed("username") {
+		patch.Username = &username
+	}
+	displayName, err := cmd.Flags().GetString("display-name")
+	if err == nil && cmd.Flags().Changed("display-name") {
+		patch.DisplayName = &displayName
+	}
+	description, err := cmd.Flags().GetString("description")
+	if err == nil && cmd.Flags().Changed("description") {
+		patch.Description = &description
+	}
+
+	bot, res := c.PatchBot(user.Id, &patch)
+	if err := res.Error; err != nil {
+		return errors.Errorf("could not update bot: %s", err)
+	}
+
+	printer.PrintT("Updated bot {{.UserId}} ({{.Username}})", bot)
+
+	return nil
+}
+
+func botListCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	orphaned, _ := cmd.Flags().GetBool("orphaned")
+	all, _ := cmd.Flags().GetBool("all")
+
+	page := 0
+	perPage := 200
+	tpl := `{{.UserId}}: {{.Username}}`
+	for {
+		var bots []*model.Bot
+		var res *model.Response
+		if all { //nolint:ifElseChain
+			bots, res = c.GetBotsIncludeDeleted(page, perPage, "")
+		} else if orphaned {
+			bots, res = c.GetBotsOrphaned(page, perPage, "")
+		} else {
+			bots, res = c.GetBots(page, perPage, "")
+		}
+		if res.Error != nil {
+			return errors.Wrap(res.Error, "Failed to fetch bots")
+		}
+
+		if len(bots) == 0 {
```

Comment:
Would it make sense to check that the list doesn't contain the maximum to avoid another request?

```suggestion
		if len(bots) < 200 {
```